.gitignore

```diff
@@ -7,6 +7,7 @@
 /t
 /tmp
 /out-tsc
+/src/environments/environment.ts
```

Comment:
@ajithmuthukumar-bc we cant add this file to .gitignore since we will need this to add new configs to UI. PLease remove. @karpagamNallasamy